### ISSUE TITLE
ERROR (MainThread) [hassio.api.proxy] Client error on WebSocket API Cannot connect to host

### ISSUE THREAD

**Issue by ciechompl** [home-assistant/supervisor#1520]
## The problem

I see error (presented below) in the System logs. 
```
20-02-19 22:15:40 ERROR (MainThread) [hassio.api.proxy] Client error on WebSocket API Cannot connect to host 172.30.32.1:8123 ssl:False [Connection refused].
```
This error is seems to be in the logs even just after clean installation not yet even configured or any plugin added. 

## Environment

- Home Assistant release with the issue: 0.105.5
- Last working Home Assistant release (if known): n/s 
- Operating environment (Hass.io/Docker/Windows/etc.): HassOS 3.11 on ESX as VM 
- Integration causing this issue: don't know
- Link to integration documentation on our website: n/a

## Problem-relevant `configuration.yaml`

```yaml

```

## Traceback/Error logs

```txt

```

**Comment by andker87** [comment]
Same here.

Hassio 0.105.5 via docker on raspbian on rpi 3b+

> ERROR (MainThread) [hassio.api.proxy] Client error on WebSocket API Cannot connect to host 172.30.32.1:8123 ssl:False [Connection refused].

**Comment by andker87** [comment]
[here](https://github.com/home-assistant/operating-system/issues/224#issuecomment-588360799) someone says that > Your addons (node-red, almond, appdaemon) cannot connect with Home Assistant when it is restarting. All of these pairs of messages are normal to see when Home Assistant Core is restarting.

**Comment by ciechompl** [comment]
but IP 172.30.32.1 is not even mine ? this is somehow internal IP within hassio so why not to eliminate such confusing error if this comes with the default package. 
not sure even which addons can it be :(

**Comment by Vingen168** [comment]
im pretty sure its "node-red-contrib-home-assistant-websocket" causing it.

untick the "I use the Home Assistant Add-on" and fill in Base URL with a access Token
the error message haven't show up for few days for me.

**Comment by fernmac2** [comment]
Same here.

**Comment by phunkybeam** [comment]
Since I have installed the update 0.106.4 on HassOS, this error also appears with me. 

So I cannot access the supervisor anymore. 

> “Unable to load the panel source: /api/hassio/app/entrypoint.js.”

What do I do now if I don't have SSH access? 

Node Red is not installed. 

> Protokolldetails ( ERROR )
> 
> Logger: homeassistant.components.hassio.http
> Integration: Hass.io (documentation, issues)
> First occured: 6. März 2020, 22:06:36 (18 occurences)
> Last logged: 19:09:05 
> 
> Client error on api app/entrypoint.js request Cannot connect to host 172.30.32.2:80 ssl:None [Connect call failed ('172.30.32.2', 80)]

[https://community.home-assistant.io/t/cant-enter-in-supervisor-unable-to-load-the-panel-source-0-106-4/176988](url)

[https://community-home-assistant-assets.s3.dualstack.us-west-2.amazonaws.com/original/3X/e/3/e33406e5c1fa4f783a82e82ac5e03e272d536ed4.jpeg](url)

**Comment by frenck** [comment]
It is indeed normal to have such an error in the logs.

When an add-on (or other system part) tries to connect to Home Assistant Core (but it isn't fully started yet), that connection would fail.

That is not a bad thing, it just reports that it happens. Connections will be picked up again when Home Assistant finished starting.

These things will happen during boot or Home Assistant Core restarts (e.g., manually invoked or during upgrade).

The `172.30.32.x` is an internal network, which Home Assistant uses for internal communications between the different elements of the ecosystem. Which is right and not weird.

I'm closing this issue, as it is not a bug, but expected behavior.

**Comment by szczepaxn** [comment]
but i am affraid that is flooding in logs all the time... and it may cause to provide a delay for any events updates or state changes

**Comment by dereitz** [comment]
This happens for me after my HASSIO has been running for a while and seems to get worse over time.  And it happens for various tabs at various times (sometimes Superviser, sometimes ESPHome, sometimes other tabs).  I'm unable to replicate reliably and a restart usually alleviates.  I'm not sure if it matters, but I used the install.sh script to install HASSIO into an existing Ubuntu 16.04 installation.

**Comment by BarrettLowe** [comment]
I agree with @dereitz - there is something going on out of the ordinary. Sometimes I can fix it by restarting my supervisor, other times I just have to reboot the whole machine. I'll try and get some logs next time it happens instead of just treating the symptom :)

**Comment by Grappa2020** [comment]
i'm facing the same problem for the last month assuming one of the update started that.
after HA restart from web GUI beside that log error also getting that message on notifications:

> Invalid config
> The following integrations and platforms could not be set up:
> homekit
> zeroconf
> ios
> default_config
> Please check your config.

restarting the Pi solve the problem

**Comment by szczepaxn** [comment]
![screenshot 63](https://user-images.githubusercontent.com/33122831/80566000-30adaa00-89f2-11ea-9e56-835db97984a8.png)

And sometimes these are endless until reboot or it connected after some random time period...
restarting supervisor (ssh) gives the same logs and pure similar behavior

**Comment by Gooman-rus** [comment]
I have the same problem on Supervisor page. Resource http://local_ip:8123/api/hassio/app/entrypoint.js returns 502 error. I can’t access the Supervisor page. Version 0.109.2.

Logs:
> Client error on api app/entrypoint.js request Cannot connect to host 172.30.32.2:80 ssl:None [Connect call failed (‘172.30.32.2’, 80)]

**Comment by Grappa2020** [comment]
Update.
Uninstalling Almond add-on solved the problem for me.

**Comment by Gooman-rus** [comment]
I have again this issue and superviser is not working.

**Comment by thundergreen** [comment]
Does homeassistant start at all? When I watch the logs of homeassistant it stops at a certain point but webinterface connection refused with errors mentionned here. Rebooting or setting I tables won't help.

I have the spervised install on ubuntu 20.04

**Comment by MickPBduece** [comment]
My situation may have started with a problem where HA would fall over after an a few minutes, some hours or even a day or two.  I tried updating Raspian, Docker etc then had issue where supervisor was not running after HA started.  I could start the container and it would run but eventually fall over.  Today I did the full-upgrade and currently supervisor will start but when I try to update supervisor it falls over.
I see a few things in the log 

I am curious why this was closed
